---
layout: post
---

I am a plant quantitative geneticits and computational biologist, and digital agriculture and robotics enthusiast, and an amateur astronomer, painter, online blogging, photography and vivid cook. I have over ten years of research experience in the field of plant breeding, quantitative genetics and high-throughput phenotyping, including, I have developed statistical models to non-destructively phenotype various seed composition using near infrared reflectance spectroscopy (NIRS) and nuclear magnetic resonance (NMR) instruments. I have developed gene specific molecular markers for marker selection studies and prepared RAD Genotype-By-Sequencing libraries for NextGen sequencing, and called SNPs using TASSEL and Sentioen pipeline, and built genetic maps using SNP and haplotype based genetic maps, and conducted joint-linkage QTL mapping and genome wide association studies. I have strong knowledge and experience in conducting big data analysis and visualization especially on a large set of phenotypic and genotypic data in software such as MS Excel, TASSEL, R and SAS and CIRCOS (written in PERL).

## Education

#### 2012–2017 PhD - Plant Breeding, Genetics & Genomics — University of Missouri Columbia, USA
•	Dissertation ***Evaluation of Teosinte Genetic Diversity for Agronomic and Domestication Traits in Maize***
Received Dr. Marcus Zuber endowment fellowship
Presented research at Maize genetics conference, Plant Animal Genetics conference and The Tri-Societies meeting (ASA, CSSA, SSSA)     International meetings
Recipient of 2017 Research in Plant Biology Award (Monetary award $750) and 2017 MU Life Sciences Week Research Recognition award (Monetary award $275)
Recipient of DPS and IPG travel awards

#### 2008–2012 BSc Agricultural Sciences — Truman State University, USA
•	Received President’s Honorary Scholarship and Dr. Max E. Bell Scholarship
•	Presented research at student research conferences at Truman, Adair County extension center and annual Maize genetics conference

## Work Experience

#### 2018–Present Computational biologist (Postdoc) — Cornell University, Geneva, NY
Analyzed of large-scale genotyping data (from Genotyping-by-Sequencing, Amplicon Sequencing, Illumina resequencing, and/or PacBio de novo sequencing) such as SNP and Haplotype calling. Developed genetic markers for traits like powdery mildew disease resistance and fruit quality, to assist breeders with marker assisted selection; conducted statistical analysis of large-scale data sets from DNA sequencing and trait data from related and unrelated vines for genetic mapping in Lep-MAP3, QTL mapping in R, genome wide association studies in TASSEL/FarmCPU/GAPIT. Managed grapevine breeding database www.VitisBase.org , and led setting up trials and phenotype data collection on Field Book android application.

#### 2017 May–2017 Dec Quantitative Geneticist (Postdoc) — University of Illinois - Urbana-Champaign, IL

Led and managed undergraduate students and technicians in the installation of over 2,000 minirhizotron tubes in field to image and study maize roots with Bartz imaging systems; and collections of minirhizotron images, moisture data, flowering, above ground biomass and yield harvest. Identified genetics basis for maize root structure and control of plant nutrient relations in Illinois Protein Strain Recombinant Inbred (IPSRI) lines of maize

#### 2012—2017 Graduate Research Assistant (PhD) - University of Missouri Columbia, USA

Involved in the development of novel genetic resource and mapping populations such as near isogenic and synthetic lines derived from domesticated corn and its wild relative, Teosinte. Conducted Joint linkage QTL analysis and Genome wide association study to elucidate several aspects of genetic diversity in maize; genetic basis of various agronomic, fitness and yield related traits such as seed quality and composition in trials conducted with multiple replications in different environment and year. Investigated gene by gene epistatic interactions for various adaptation and domestication related traits in trials with multiple replications and environment.

#### 2009-2012 Undergraduate Research Assistant (BSc) - Truman State University, USA
Conducted molecular genetic analysis to exploit the biodiversity of exotic inbred lines from Central and South America to develop high amylose corn using SSR markers and gel electrophoresis. Assisted in preparing summer maize nurseries such as plowing field with a tractor, planting seeds with hand and dumping seed in seed planter, pollinating and collecting various agronomic data from field trials, harvesting and analyzing data. Developed gene specific molecular markers to select alleles responsible for high amylose in maize kernels with better yield, which I validated their stability in trials conducted in multiple replications in different environments. Wrote annual USDA-ARS Germplasm Enhancement of Maize (GEM) report and presented at ASTA meeting in Chicago, Illinois 

#### 2008-2012 Cafeteria student worker - Sodexo Inc. 
Served and stocked food as well as performed all the cleaning related activities in the cafeteria.

#### 2011-2012 Comparative genomics researcher - Truman State University, USA
Analyzed retained homeologous paralog pairs from syntenic regions in Maize and Sorghum. Visualized genomic data of approximately 1Mb in size in CIRCOS, and conducted comparative genetic study in online tools such as GeVo function of CoGe, Annotated genes using InterProScan and expression analysis data from Plexdb.org 

#### Summer 2009 & 2012 NSF TruScholars Research Program - Truman State University, USA
Designed gene specific molecular marker for a novel of starch branching enzyme Ia (SBE-1a) allele involved in high amylose starch in maize. Discovered novel recessive starch branching enzyme 1a (sbe1a) likely plays a significant role in the presence of the ae allele in elevating starch amylose from ~55% to >70% in the germplasm release 'GEMS-0067'. Conducted linkage based mapping studies to identify QTLs for high amylose starch in maize using SSR markers and QTL cartographer software. Sequenced starch branching enzyme allele (SBE1a) from high amylose maize line, GEMS-0067; Discovered 8bp insertion site at intron 9 resulting splicing mutation, which putatively resulted in truncated protein product post-translation.

## Publications

#### A rhAmpSeq haplotype strategy targeting the collinear core genome improves marker transferability across a diverse genus
Cheng Zou, **Avinash Karn**, Bruce Reisch, Allen Nguyen, Yongming Sun, Yun Bao, Michael S. Campbell, Deanna Church, Stephen Williams, Timothy P. L. Smith, Anne Fennell, Doreen Ware, Jason Londo, Qi Sun and Lance Cadle-Davidson
(2018) (In review)
*Nature Biotechnology*

#### Computational analysis of AmpSeq data for targeted, high-throughput genotyping of amplicons 
Jonathan Fresnedo-Ramírez , Shanshan Yang , Qi Sun, **Avinash Karn** , Bruce I. Reisch, Lance Cadle-Davidson (2018) (Accepted)
*Frontiers in Plant Sciences*

#### Genome-Wide Association Study (GWAS) and Linkage Mapping Reveal a Major QTL Underlying Acylated Anthocyanin Levels in Grapevine Hybrid Families
**Avinash Karn**, Shanshan Yang, Cheng Zou, Fred Gouker, Jonathan Fresnedo-Ramírez, Qi Sun, Jason Londo, Bruce I. Reisch, Gavin Sacks, and Lance Cadle-Davidson  (In preparation)

#### Genetic Analysis of Teosinte Alleles for Kernel Composition Traits in Maize 
**Karn, Avinash**, Gillman, J. D., & Flint-Garcia, S. A. (2017) 
*G3: Genes, Genomes, Genetics*

#### Development of Rigorous Fatty Acid Near-Infrared Spectroscopy Quantitation Methods in Support of Soybean Oil Improvement.
**Karn, Avinash**, Heim, C., Flint-Garcia, S., Bilyeu, K., & Gillman, J. (2016)
*Journal of the American Oil Chemists*

#### Breeding Specialty Starch Maize Using Exotic Genetic Resources for Gene Discovery of Novel Alleles and Modifiers with Materials Generated from the USDA-ARS GEM Project  (2011)
**Karn, Avinash**, Brent Buckner, Diane Janick-Buckner, Mark Campbell, Michael Blanco, Marvin Scott, and Adrienne Moran Lauter .
*Maize Genetics Conference Abstracts*

## Knowledge

#### Programming languages

R, SAS, PERL, HTML, MS Command prompt and Excel.

#### Operating systems

Linux and Windows.


<section id="contact">
    <div class="container">
        <div class="row">
            <div class="col-lg-8 col-lg-offset-2 text-center">
                <h2 class="section-heading">Let's Get In Touch!</h2>
                <hr class="primary">
                <p>If you have any question, suggestion or comment, please contact me by</p>
            </div>
            
        <div class="col-lg-2 col-lg-offset-1 text-center">
            <a href="https://www.linkedin.com/in/avikarn/" target="_blank">
                <i class="fa fa-linkedin fa-3x wow bounceIn" data-wow-delay=".1s"></i>
                <p>LinkedIn</p>
            </a>
        </div>
            
        <div class="col-lg-2 col-lg-offset-1 text-center">
            <a href="https://twitter.com/avi_karn/" target="_blank">
                <i class="fa fa-twitter fa-3x wow bounceIn" data-wow-delay=".1s"></i>
                <p>Tweet</p>
            </a>
        </div>
  
        <div class="col-lg-2 col-lg-offset-1 text-center">
            <a href="mailto:akarn@illinois.edu">
            <i class="fa fa-envelope-o fa-3x wow bounceIn" data-wow-delay=".1s"></i>
                <p>Email</p>
            </a>
        </div>
       
        <div class="col-lg-2 col-lg-offset-1 text-center">
            <a href="https://www.facebook.com/avi.karn" target="_blank">
            <i class="fa fa-facebook fa-3x wow bounceIn" data-wow-delay=".1s"></i>
                <p>Facebook</p>
            </a>
        </div>
        

    </div>
</section>

<p><script type="text/javascript" src="//ra.revolvermaps.com/0/0/6.js?i=0rn8vq73z9x&amp;m=7&amp;c=ff0000&amp;cr1=ffffff&amp;f=arial&amp;l=1" async="async"></script></p>


<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-123359651-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-123359651-1');
</script>
